docs/technical/auth-config.md

```diff
@@ -25,11 +25,48 @@ The JSON file contains the following fields:
                 The auth mode to use. Not case-sensitive.<br/>Available options:
                 <ul>
                     <li><code>basic</code></li>
+                    <li><code>plugin</code></li>
                 </ul>
             </td>
         </tr>
         <tr>
-            <td><code>parameters</code></td>
+            <td><code>path</code> (<code>plugin</code> mode only)</td>
```

Comment:
Interesting that these are not part of `parameters` for `plugin` mode. Could we make that swap? It can have a `pluginParameters` value inside if more customization is needed.

Reply:
My initial thought was "we have a few named parameters that apply to every mode, and anything else fits in `parameters`." If you anticipate that `basic` is the outlier and really everyone will be using a plugin, then having these values where they are makes more sense.